app/lib/submission_builder/ty2024/states/md/md_return_xml.rb

```diff
@@ -41,7 +47,8 @@ def form1099g_builder
           def supported_documents
             calculated_fields = @submission.data_source.tax_calculator.calculate
             has_income_from_taxable_pensions_iras_annuities = calculated_fields.fetch(:MD502_LINE_1D)&.to_i.positive?
-            has_md_su_subtractions = calculated_fields.fetch(:MD502_SU_LINE_1).positive?
+            has_md_su_subtractions = calculated_fields.fetch(:MD502_LINE_13).positive? || form_has_non_zero_amounts("MD502_SU_", calculated_fields)
```

Comment:
can `:MD502_LINE_13` even be positive if `form_has_non_zero_amounts("MD502_SU_", calculated_fields)` is true? Just wondering if only the one check is necessary. Same for below.

Reply:
yeah i asked erica and i guess the condition for both 502SU and 502CR is that either:
1) the given line from the 502 has a value
OR
2) the attached form itself has any values
i'm not sure what the odds are that one would be true and the other false, but i did clarify that we want to check both